app/views/admin/edition_tags/edit.html.erb

```diff
@@ -30,7 +30,10 @@
       margin_bottom: 2,
     } %>
 
-    <%= form_for @tag_form, url: admin_edition_tags_path(@edition), method: :put do |form| %>
+    <%= form_for @tag_form, url: admin_edition_tags_path(@edition), method: :put, data: {
+      module: "track-selected-taxons",
+      "request-path": request.path,
```

Comment:
Do we _need_ to pass this through from the server side? 🤔

Unless I'm mistaken, I think `request.path` in Rails will give the same result as `window.location.pathname` in JavaScript.

I know you're just replicating existing behaviour, so no need to change if you want to keep it strictly like-for-like.

Reply:
good point - we could actually, I think I went too much into isolation principles and wasn't trying to use anything other than the module scope.

Reply:
updated